OpenSim/Simulation/Model/ForceSet.cpp

```diff
@@ -281,15 +286,16 @@ getStateVariableNames(OpenSim::Array<std::string> &rNames) const
 bool ForceSet::
 check() const
 {
-    bool status=true;
+    bool status = true;
 
     // LOOP THROUGH ACTUATORS
-    ScalarActuator *act;
-    int size = getSize();
-    for(int i=0;i<size;i++) {
-        act = dynamic_cast<ScalarActuator *>(&get(i));
-        if(!act) continue;
+    for (Force& f : *this) {
+        auto* act = dynamic_cast<ScalarActuator*>(&f);
+        if (!act) {
+            continue;
+        }
     }
+    // TODO: this is a bogus method?
 
-    return(status);
+    return status;
```

Comment:
This is bogus because it effectively has no side-effects and just returns `true`